src/main/java/htsjdk/samtools/DuplicateScoringStrategy.java

```diff
@@ -33,9 +35,13 @@
 
     public enum ScoringStrategy {
         SUM_OF_BASE_QUALITIES,
-        TOTAL_MAPPED_REFERENCE_LENGTH
+        TOTAL_MAPPED_REFERENCE_LENGTH,
+        RANDOM,
     }
 
+    /** Hash used for the RANDOM scoring strategy. */
+    private static Murmur3 hasher = new Murmur3(1);
```

Comment:
Done.
> On Aug 19, 2016, at 2:53 PM, Yossi Farjoun <notifications@github.com> wrote:
> 
> In src/main/java/htsjdk/samtools/DuplicateScoringStrategy.java <https://github.com/samtools/htsjdk/pull/688#discussion_r75534108>:
> 
> >      }
> >  
> > +    /** Hash used for the RANDOM scoring strategy. */
> > +    private static Murmur3 hasher = new Murmur3(1);
> final?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/samtools/htsjdk/pull/688/files/1a5616268a3d8f0a69a734a7ab26a7ca3ef4f0b2#r75534108>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABiN-kQGAN0-1Z0BymEovtIM2v79A87Uks5qhfurgaJpZM4Joo0U>.
>